gateway/mw_jwt.go

```diff
@@ -792,7 +792,10 @@ func (k *JWTMiddleware) ProcessRequest(w http.ResponseWriter, r *http.Request, _
 	k.reportLoginFailure(tykId, r)
 	if err != nil {
 		logger.WithError(err).Error("JWT validation error")
-		return errors.New("Key not authorized:Unexpected signing method."), http.StatusForbidden
+		errorDetails := strings.Split(err.Error(), ":")
+		if errorDetails[0] == "Unexpected signing method" {
```

Comment:
can we move this to a constant? Like here: https://github.com/TykTechnologies/tyk/blob/0ed167f12f96fb684d369df4eb72823d30b74943/gateway/handler_error.go#L31

Reply:
we want here generic message, but yes we can use constant, btw do we have general approach about how we want to use text constants? maybe in future we want to have full language support with translations etc..